package.json

```diff
@@ -57,8 +56,13 @@
     "redux"
   ],
   "dependencies": {
+    "broccoli-merge-trees": "^1.1.5",
     "ember-cli-babel": "^5.1.7"
   },
+  "peerDependencies": {
```

Comment:
Are peerDependencies required? just curious as I often avoid them when using npm 3+

also - did you run into any issues with redux-saga or was adding the npm dep above good enough?